apps/webapp/app/utils.ts

```diff
@@ -35,8 +35,8 @@ export function safeRedirect(
 export function useMatchesData(
   id: string | string[],
   debug: boolean = false
-): RouteMatch | undefined {
-  const matchingRoutes = useMatches();
+) {
+  const matchingRoutes = useMatches() as UIMatch<unknown, Handle>[];
```

Comment:
Why is this being cast?

Reply:
Hi @ericallam  with the remix update the RouteMatch Type has been removed so we have to use the UIMatch<unknown, unkown> type as a replacement for that.
Since useMatches hook returns UIMatch<unknown, unkown>[] as return type, we won't get type completions and the below paths.reduce function is giving type errors